recipes/scikit-build/python27-x64.patch

```diff
@@ -0,0 +1,13 @@
+MSVC 9 x64 is broken on appveyor. Use NMake there.
```

Comment:
It could easily be fixed using the `patch_vs2008` add-on provided by `scikit-ci-addons`

See http://scikit-ci-addons.readthedocs.io/en/latest/addons.html#patch-vs2008-py